File: DaySix/Java/DateOfBirth.java
import java.util.Scanner;
import java.time.LocalDate;
public class DateOfBirth {

   public static void main(String [] args) {
   Scanner scanner = new Scanner(System.in);

   System.out.println("Welcome to your age reveal");

   System.out.println("Kindly enter your date of birth in this format.....D/M/Y");

   System.out.print("Enter your name: ");
   String name = scanner.nextLine();

   System.out.print("Enter your date of birth: ");
   String dateOfBirth = scanner.nextLine();

   int age = calculateAge(dateOfBirth);

   System.out.println(name + " you are " + age +  " years old");

    }

   public static int calculateAge(String dateOfBirth) {
   String [] parts = dateOfBirth.split("/");
   int birthYear = Integer.parseInt(parts[2]);
   int currentYear = 2024;
   return currentYear - birthYear;
     }

 }

File: DaySix/Python/date_of_birth.py
from datetime import datetime

def calculate_age(date_of_birth):
	parts = date_of_birth.split("/")
	birth_year = int(parts[2])
	current_year = 2024
	return current_year - birth_year


print("Welcome to your age reveal")

print("Kindly enter your date of birth in this format.....D/M/Y")

name = input("What is your name?: ")

date_of_birth = input("Enter your date of birth: ")

age = calculate_age(date_of_birth)

print(name, "You are", age, "years old")


